bundles/org.openhab.binding.dbquery/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dbquery"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="query">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="influxdb2"/>
+			<bridge-type-ref id="jdbc"/>
+		</supported-bridge-type-refs>
+		<label>Query Thing</label>
+		<description>Thing that represents a native query</description>
+
+		<channels>
+			<channel id="execute" typeId="execute-channel"/>
+			<channel id="resultString" typeId="result-channel-string"/>
+			<channel id="resultNumber" typeId="result-channel-number"/>
+			<channel id="resultDateTime" typeId="result-channel-datetime"/>
+			<channel id="resultContact" typeId="result-channel-contact"/>
+			<channel id="resultSwitch" typeId="result-channel-switch"/>
+
+			<channel id="parameters" typeId="parameters-channel"/>
+			<channel id="correct" typeId="correct-channel"/>
+			<channel id="calculateParameters" typeId="calculate-parameters-channel"/>
+		</channels>
+
+
+
+		<config-description>
+			<parameter name="query" type="text" required="true">
+				<label>Query Definition</label>
+				<description>Query definition using native query language</description>
+				<context>script</context>
+			</parameter>
+			<parameter name="hasParameters" type="boolean">
+				<label>Query has Parameters</label>
+				<description>True if the query has parameters, otherwise false</description>
+				<default>false</default>
+			</parameter>
+			<parameter name="scalarResult" type="boolean">
+				<label>Scalar result</label>
+				<description>True if the query always return only one single scalar value (only one row and one value-column in this
+					row), otherwise false</description>
+				<default>true</default>
+			</parameter>
+			<parameter name="scalarColumn" type="text" required="false">
+				<label>Scalar column name</label>
+				<description>The column's name that is used to extract scalarResult. If only one column is returned this
+					parameter
+					can be blank</description>
+			</parameter>
+			<parameter name="interval" type="integer" min="0">
+				<label>Interval</label>
+				<description>
+					An interval, in seconds, the query will be repeatedly executed. Default values is 0, which means that
+					query is never executed manually (you need to send ON to execute channel)
```

Comment:
```suggestion
					query is never executed automatically. You need to send the ON command each time you wish to execute.
```

Does this make more sense, or did I get it wrong?

Reply:
Yes it's absolutely correct.